plugins/outputs/influxdb/http.go

```diff
@@ -209,6 +209,12 @@ func (c *httpClient) CreateDatabase(ctx context.Context, database string) error
 
 	resp, err := c.client.Do(req.WithContext(ctx))
 	if err != nil {
+		// Close connection after a timeout error. If this is a HTTP2
```

Comment:
This is copy pasted in three places.  That's enough to make a new function worth the trouble.

func (c *httpClient) handleTimeout(err error)